### PR TITLE
Add AA+ and AAA+ text 

### LINKED ISSUE #86: Add the missing AA+ and AAA+ texts
#### In the `index.html` file:
\- Within each `div` with the `.check-result` class and just after the `h3` element add a new `div` with the class of 
`.level-labels`.

\- Within each of the new `.level-labels` add two spans, one with the text `AA+` and the other with the text `AAA+`. Give each one a class of `.level`. (Only add one `span` for the last `.level-labels` element, with the text `AA+`).

\- Within each `.check-result`, wrap all the `div`s with the class of `.pass` or `.fail` in a `div` with a class of `icons`.

#### In the `styles.css` file:
\- Remove the `margin-right` property from the `.ratio-result` class.
\- Add the following styles to the `.ratio-result` class:
```
.ratio-result {
  margin: 0 2rem;
  padding: 1rem;
}
```
\- Give the `.level-labels` the following styles:
```
.level-labels {
  margin: 1.5rem 0 1rem 0;
  display: flex;
  justify-content: space-around;
  font-variation-settings: "wght" 600;
}
```
\- Set each `.level`'s width to `3rem`:
```
.level {
  width: 3rem;
}
```
\- Add the following styles to the `.icons` class:
```
.icons {
  margin: 1rem 0;
  display: flex;
  justify-content: space-around;
}
```
\- Modify the `large` class to have the following styles:
```
  font-size: 2.5rem;
  font-variation-settings: "wght" 650;
```

### PR DESCRIPTION
**Pull request by Okibe00**
# Description

Added the AA+ and AAA+ text to index.html,  modifications to `.large` CSS selector and  refactor of index.html to include a `.levels`, `.level-label`, `.icons` and styling of these classes in style.css.

Fixes #86 

## Type of change
- [x] Chore items (this includes basic clean up of files, or updates to packages)

## Checklist

- [x] I have a descriptive title for my PR
- [x] I have linked this PR to the corresponding issue. [See how to do that here.](https://docs.github.com/en/issues/tracking-your-work-with-issues/linking-a-pull-request-to-an-issue)
- [x] I have run the project locally and reviewed the code changes
- [x] My changes generate no new warnings

### CODE REVIEW

**Review by AndreyDmitriev39r — APPROVED** [review]
This is good to go.

File: styles.css [hunk 1]
```diff
@@ -90,10 +90,8 @@ footer p {
   margin-top: -10px;
 }
 .large {
```
Comment (sparkO7) [inline styles.css:92]: One little change here, which is of course not your fault. I forgot to mention to leave the `margin` property.
So, please add the `margin: 0` to the `.large` class.

File: styles.css [hunk 2]
```diff
@@ -127,9 +125,27 @@ footer p {
...
+  justify-content: space-around;
+  font-variation-settings: "wght" 600;
+}
+
+.level {
+  width: 3rem;
+}
+
+.icons {
+  width: 110px;
```
Comment (sparkO7) [inline styles.css:144]: I understand why you've added the `width` here. But, don't worry about it, you can remove it. Issue #85 will have a fix for this.

**Review by sparkO7 — CHANGES_REQUESTED** [review]
Apart from the two small fixes, everything LGTM 👍

File: styles.css (hunk 2 above)
Comment (Okibe00) [reply styles.css:144]: Ohh and to think i wanted  asking before making the change.Alright then.

Comment (Okibe00) [reply styles.css]: changes have been made please have a look.

**Pull request merged**